modules/web/src/app/settings/admin/defaults/template.html

```diff
@@ -180,7 +260,7 @@
   </mat-card-content>
 </mat-card>
 
-<mat-card>
+<mat-card *ngIf="interfaceTypeUrl === adminPanelView.Limits">
```

Comment:
![iss02](https://user-images.githubusercontent.com/17727069/204742987-2c3598da-a062-4537-b3c3-f310106acab7.png)


This layout can be discussed with @cschieder